geoportailv3/static/js/locationinfo/locationinfodirective.js

```diff
@@ -111,10 +111,16 @@ app.LocationinfoController =
     }
     if (newVal === false) {
       this.stateManager_.updateState({'crosshair': false});
+      var mapCenterCoordinate = this['map'].getView().getCenter();
+      if (goog.isDefAndNotNull(mapCenterCoordinate)) {
```

Comment:
`getCenter`'s return type is `ol.Coordinate|undefined`. But given the way we use the View in the application `getCenter` should never return `null` or `undefined`. An assertion is therefore the best thing to use here.

```js
goog.asserts.assert(goog.isDefAndNotNull(viewCenter));
```